jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullAndCacheBaseImageLayerStep.java

```diff
@@ -36,14 +38,48 @@
 import javax.annotation.Nullable;
 
 /** Pulls and caches a single base image layer. */
-class PullAndCacheBaseImageLayerStep implements Callable<CachedLayerAndName> {
+class PullAndCacheBaseImageLayerStep implements Callable<PreparedLayer> {
```

Comment:
Can we come up with a different name for this step, since it may not actually pull-and-cache the given image layer?  `ObtainBaseImageLayerStep`?